src/cache_dit/serve/model_manager.py

```diff
@@ -178,6 +184,38 @@ def load_model(self):
                 quantization_config=quantization_config,
             )
 
+        if self.lora_path is not None and self.lora_name is not None:
```

Comment:
Many popular open-source loras require the use of specific schedulers. I think we should be consistent with their official cases to ensure that we can get the best results. For exampels, Qwen-Image-Lightning: 

https://github.com/vipshop/cache-dit/blob/08253c70709498fdd44200340e1d6fc2f48ab440/examples/registers.py#L212